cpp/include/cudf_test/column_utilities.hpp

```diff
@@ -27,13 +27,29 @@
 
 namespace cudf {
 namespace test {
+
+/**
+ * @brief Verbosity level of output from column and table comparison functions.
+ *
+ */
+enum class debug_output_level {
+  FIRST_ERROR = 0,  // print first error only
+  ALL_ERRORS,       // print all errors
+  QUIET,            // no debug output
```

Comment:
I'm not sure if this colon causes compiling warning or error.
```suggestion
  QUIET            // no debug output
```